<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildPage</class>
 <widget class="QWidget" name="BuildPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="te_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>801</width>
     <height>321</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_model">
      <property name="text">
       <string>当前android对应工程型号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_model">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lbl_lunch_mode">
      <property name="text">
       <string>工程版本:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_lunch_mode"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>737</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="ckb_load_prj">
        <property name="text">
         <string>导入工程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckb_kernel">
        <property name="text">
         <string>编译内核</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckb_android">
        <property name="text">
         <string>编译安卓</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckb_fls">
        <property name="text">
         <string>生成fls烧录固件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckb_ota">
        <property name="text">
         <string>生成ota整包</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_thread_num">
        <property name="text">
         <string>编译线程数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_thread_num"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_build">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
